src/test/java/teammates/test/cases/action/InstructorFeedbackEditCopyActionTest.java

```diff
@@ -268,16 +268,16 @@ public void testExecuteAndPostProcess() {
 
         assertEquals("", editCopyData.redirectUrl);
 
-        expectedString = "\"\" is not acceptable to TEAMMATES as a/an feedback session name because it is empty. "
+        expectedString = "The field feedback session name is empty. "
```

Comment:
I think the issue is applicable only where the field is empty because for other errors we are still using this [`"Invalid name" is not acceptable to TEAMMATES as a/an feedback session name because it contains invalid character."`] as an error statement and I think we don't need to highlight the field in this case.